proto/api.proto

```diff
@@ -143,6 +143,14 @@ message SimOptions {
 	bool use_labeled_rands = 9; // Use test level RNG.
 }
 
+enum SpellType {
+	SpellTypeUnknown = 0;
+	SpellTypeCast = 1;
+	SpellTypePeriodic = 2;
+	// Needed for Frontend
+	SpellTypeAll = 3;
```

Comment:
What exactly is the use case here for SpellTypeAll? Combing both doesn't make much sense to me, maybe this could just be  `bool IsPeriodic`?

Reply:
I think this is now redundant since I split Hits and Ticks thus merged spells no longer need this